client/app/application.coffee

```diff
@@ -58,10 +47,12 @@ class App extends Application
     # Backbone Router
     initializeRouter: () =>
         @router = new Router app: @
-        @router.route \
-            'register(/:step)',
-            'register',
-            @handleRegisterRoute
+        # if onboarding, the pathname will be '/register*'
+        if window.location.pathname.indexOf('/register') is 0
```

Comment:
No need here to test window.location, this mechanism, is already handled by the router.

As we add the `register` route, the `@initializeOnboarding` method will be automatically and transparently called.